lib/cdo/aws/cloudfront.rb

```diff
@@ -258,6 +260,10 @@ def self.config(app, reference = nil)
               s3_origin_config: {
                 origin_access_identity: ''
               },
+              custom_origin_config: {
+                origin_read_timeout: 5,
+                origin_keepalive_timeout: 30
+              },
```

Comment:
these lines need to be removed - only one of `s3_origin_config` _or_ `custom_origin_config` are required.